wrapper/python/wolfcrypt/wolfcrypt/random.py

```diff
@@ -46,23 +46,23 @@ def byte(self):
         """
         Generate and return a random byte.
         """
-        result = t2b("\0")
+        result = _ffi.new('byte[1]')
 
         ret = _lib.wc_RNG_GenerateByte(self.native_object, result)
         if ret < 0:
             raise WolfCryptError("RNG generate byte error (%d)" % ret)
 
-        return result
+        return _ffi.buffer(result, 1)[:]
```

Comment:
Hi @embray,

why did you choose `_ffi.buffer(result, 1)[:]` over `_ffi.buffer(result)[:]`?